mobile-browser-based-version/src/helpers/training_script/training_manager.js

```diff
@@ -117,6 +118,13 @@ export class TrainingManager {
         );
     }
 
+    saveWorkingModel(){
```

Comment:
Perfect, I think that loading/storing model for each function will be the new way of handling models throughout the entire application.